src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,107 @@ The accordion component lets users show and hide sections of related content on
+If you decide to use one of these components, consider:
+- if the user needs to look at more than one section at a time — an accordion can show multiple sections at a time, unlike tabs
+- if the user needs to switch quickly between sections — tabs can show content without pushing other sections down the page, unlike accordions
+- if there are many sections of content — accordions can fit more sections as they’re arranged vertically, unlike tabs which are arranged horizontally
+- if there’s only one or two pieces of short, less important content — the details component is more suitable as it’s visually smaller and less prominent than an accordion or tabs
 
 ## How it works
 
 There are 2 ways to use the accordion component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
-### Find out how your users interact with the content
+### Section heading buttons
+
+An accordion will usually start with all sections hidden. To show a section, the user can interact anywhere in the heading button.
+
+The heading button includes all of these areas:
+- heading text
+- summary text (if you decide to add one)
+- call-to-action text to 'show' or 'hide'
+
+For users of screen readers, all the text in the button will be read as a single statement (separated by commas to allow for slight pauses). There’s also some visually hidden content in the heading text to help announce the call-to-action as 'show this section' or 'hide this section'
 
-Users might need the sections they have opened to stay open if they leave and then return to the page. You can configure the accordion component to stay open.
+[EXAMPLE OF ACCORDION, GENERAL]
 
-Do user research to find out the best order for the sections.
+#### Write clear button text
 
-### Use clear labels
+Write the heading and summary text like any other button text and: 
+- use sentence case
+- describe the content that will show
+- keep it short
 
-Accordions hide content, so the labels need to be clear. If necessary, you can add a summary line to help users understand what is in the section.
+Users of screen readers will find it difficult to navigate the accordion if the button text is too long.
 
-{{ example({group: "components", item: "accordion", example: "with-summary-section", html: true, nunjucks: true, open: false, size: "xl"}) }}
+If you struggle to come up with clear button text, it might be because the way you’ve separated the content is not clear. Organise sections in a way that makes sense to users, based on their needs.
 
-If you struggle to come up with clear labels, it might be because the way you’ve separated the content is not clear.
+#### Adding a summary line
```

Comment:
@calvin-lau-sig7 This section is coming on well 👍 

However, it doesn't contain clear call to actions _if you're already using the summary line_. I'm currently reviewing the release note and was hoping that we could have something like the following in the release note:
> Review the [guidance](https://deploy-preview-1961--govuk-design-system-preview.netlify.app/components/accordion/#adding-a-summary-line) to see if you need to change your summary line content

And then for this section in the guidance to contain something like the following calls to action:

> If you are already using the summary line in your accordions, you should review whether the summary line is necessary or whether you should make the summary line shorter.

OR 

Maybe we should decide that any changes teams need to make should be covered in the release note and put the above calls to action there. This would duplicate the guidance though 🤔 

(The name of this section also doesn't cover the use case where a team is already using one)

Reply:
As discussed, we're putting the calls to action in the release notes, pointing to this guidance.